app/templates/components/forms/orders/order-form.hbs

```diff
@@ -114,6 +114,7 @@
           <label class="required">{{t 'Select country'}}</label>
           {{#ui-dropdown class="search selection" selected=data.country
                         onChange=(action (mut data.country)) as |execute mapper|}}
+            {{input type='hidden' id='country' value=data.language}}
```

Comment:
Why data.language? Shouldn't it be data.country? @kushthedude

Reply:
Missed it ,dont know how😬